pkg/skaffold/event/server.go

```diff
@@ -62,14 +65,19 @@ func (s *server) Handle(ctx context.Context, event *proto.Event) (*empty.Empty,
 }
 
 // newStatusServer creates the grpc server for serving the state and event log.
-func newStatusServer(port string) (func() error, error) {
-	if port == "" {
+func newStatusServer(originalPort int) (func() error, error) {
+	if originalPort == -1 {
```

Comment:
I don't see why this would be -1? i see its called here with default port 50051?https://github.com/GoogleContainerTools/skaffold/blob/80d034707f40bfae84370ac984747e623a605853/pkg/skaffold/event/event.go#L103

Reply:
this was for testing as a way to bypass creating an RPC server in some cases. I probably should have removed it.